src/Helpers/DownloadReport.js

```diff
@@ -74,7 +95,13 @@ class DownloadReport  {
             let data = format === 'json' ? JSON.stringify(response) : response;
 
             downloadFile(data, this.filename(page), format);
+
+            clearNotification();
```

Comment:
I'm a bit skeptical about `            clearNotification();` it looks we call  `            clearNotification();`  repetitively, how can we make use addNotification more abstract to handle the cases where we have to clear the notification?